commands/build.go

```diff
@@ -81,10 +81,9 @@ type commonOptions struct {
 	progress     string
 	pull         *bool
 
-	// golangci-lint#826
```

Comment:
Looks to link to https://github.com/golangci/golangci-lint/issues/826 (about this linter not being good)